javascript/modal-operations.js

```diff
@@ -48,6 +48,7 @@ window.addEventListener("load", () => {
 
 const showDeletionModal = (e) => {
     $("overlay").classList.remove("flex");
+    $("overlay").classList.remove("overlay-show");
     $("floating-button-overlay").classList.remove("modal-backdrop");
```

Comment:
This is causing the backdrop to vanish immediately, even before the modal finishes it's animation and exits.

Make sure that backdrop disappears only **after** the modal has completely gone off the screen.